portal/templates/contract/components/issue-or-delete-facility-link.njk

```diff
@@ -27,8 +27,11 @@
       <a href="/contract/{{ dealId }}/submission-details#{{ facilityName }}-{{ facility._id }}" class="govuk-link" data-cy="{{ facilityName }}-issue-facility-{{ facility._id }}">Facility issued</a>
 
     {% elseif (userIsMaker and dealIsEditable) and (dealStatus === "Draft" or dealStatus === "Further Maker's input required") and not dealSubmissionDate %}
-      <a href="/contract/{{ dealId }}/{{ facilityName }}/{{ facility._id}}/delete" class="govuk-link" data-cy="{{ facilityName }}-delete-{{ facility._id }}" aria-label="Delete {{ facilityName }} {{ facilityTableIndex }}">Delete</a>
-
+      {% if facility.name %}
+        <a href="/contract/{{ dealId }}/{{ facilityName }}/{{ facility._id}}/delete" class="govuk-link" data-cy="{{ facilityName }}-delete-{{ facility._id }}" aria-label="Delete {{ facilityName }} {{ facility.name }}">Delete {{ facility.name }}</a>
```

Comment:
what is the difference between `facilityName` and `facility.name` ?

Reply:
facityName is the type, like loan or bond, but facility.name is the name set to the facility (not always set)